.travis.yml

```diff
@@ -39,7 +39,10 @@ script:
   # Run on Firefox off the grid
   # Currently (Aug-2017) only seems to work with 1 thread on travis
   - mvn verify -B -Dthreads=1 -Dbrowser=firefox -Dwebdriver.gecko.driver=/tmp/geckodriver 2> /dev/null
-  
+
+  # Run on Firefox off the grid by reusing the same browser
+  - mvn verify -B -Dthreads=1 -Dbrowser=firefox -DreuseBrowser=true -Dwebdriver.gecko.driver=/tmp/geckodriver 2> /dev/null
```

Comment:
I think this needs testing with at least 2 threads

Reply:
I agree and have been testing locally with 3 threads. However, the comment right above reads "# Currently (Aug-2017) only seems to work with 1 thread on travis"